user/fonts-monaspace-otf/template.py

```diff
@@ -0,0 +1,19 @@
+pkgname = "fonts-monaspace-otf"
+pkgver = "1.101"
+pkgrel = 0
+pkgdesc = "Github Monaspace Fonts"
+maintainer = "BarryLhm <Lhm13144572810@outlook.com>"
+license = "OFL-1.1"
+url = "https://github.com/githubnext/monaspace"
+source = f"{url}/archive/refs/tags/v{pkgver}.zip"
+sha256 = "89006779394fb251a046ccaa23b875b1edced3687f1628b37c6418d8f5da63c5"
+
+
+def install(self):
```

Comment:
since you are defining `install` manually, there is no need for a separate `post_install`, just merge them together

Reply:
I have little experience in packaging and python programming...
this template is copied modified from fonts-noto